Microsoft.Toolkit.Uwp.UI.Media/Effects/BackdropEffect.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp.UI.Media.Effects.Interfaces;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Effects
+{
+    /// <summary>
+    /// A backdrop effect that can sample from a specified source
+    /// </summary>
+    public sealed class BackdropEffect : IPipelineEffect
```

Comment:
Do these all have to be sealed?  It's generally more dev-friendly if they can be extended if needed by a developer.

Reply:
They don't need to be `sealed`, though they're type matched by the [`PipelineBrush`](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/f3a4a3df76ea4dffc35fd0e9fe5c116370eb3f3c/Microsoft.Toolkit.Uwp.UI.Media/Brushes/PipelineBrush.cs), so I figured there was no point in letting users extend those classes anyway. I mean, they're basically just proxies/placeholders to let users declare those effects from XAML, they don't have any functionality per se. So I just made them `sealed` to also signal to users that there would be no point in extending those anyway. I'm open to suggestions though, if you have ideas on why leaving them opened could be useful for users, I'm all ears! 😊